Doc/whatsnew/3.11.rst

```diff
@@ -258,6 +257,12 @@ sqlite3
   threading mode the underlying SQLite library has been compiled with.
   (Contributed by Erlend E. Aasland in :issue:`45613`.)
 
+* :mod:`sqlite3` C callbacks now use unraisable exceptions if callback
+  tracebacks are enabled. Users can now register an
+  :meth:`unraisable hook handler <sys.unraisablehook>` to improve their debug
```

Comment:
```suggestion
  :func:`sys.unraisablehook` to improve their debug
```
?

Reply:
I think I prefer to keep the alt text instead of the function name in the rendered output. I've changed the ref to be `:func:` though; thanks!